services/sky/document_view.cc

```diff
@@ -171,11 +201,15 @@ mojo::ScopedMessagePipeHandle DocumentView::TakeRootBundleHandle() {
 }
 
 mojo::ScopedMessagePipeHandle DocumentView::TakeServicesProvidedToEmbedder() {
-  return services_provided_to_embedder_.PassMessagePipe();
+  // TODO(jeffbrown): Stubbed out until we migrate from native viewport
+  // to a new view system that supports embedding again.
+  return mojo::ScopedMessagePipeHandle().Pass();
```

Comment:
I believe this type should support just saying 'return nullptr;' here. if not we should fix it.  you definitely don't need the .Pass()